src/utils/routes/genericController.js

```diff
@@ -1,12 +1,12 @@
-import permissionValidator from '../validators';
+import permissionValidator from '../../validators';
 
 const genericController = (permission, controller, controllerMethod) => async (
```

Comment:
let's call genericController a `dispatcher` as it basically dispatches different actions when route middleware fires (validation, then controller)